### PR TITLE
Add's two japanese breakfast staples to breakfast category 

### PR DESCRIPTION
**Pull request by carshalash**
## About The Pull Request

I was looking at a counter full of eggs while working the bar and saw a Japanese tourist bot, I figured expanding the breakfast menu with Japanese breakfast food would be neat.  I was surprised we didn't have grilled fish though, I should probably do something about that in the future.

## Why It's Good For The Game

Gives chefs something relatively easy to make besides EGG every shift (specifically the boiled rice part), also give incentives to make soy products early on in shifts.

## Changelog
:cl:
qol: Breakfast is less eggy. Boiled rice and Milo soup are both breakfast items.
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
Huh

**Comment by tralezab** [comment]
Cool!

File: code/game/objects/items/food/soup.dm [hunk 1]
```diff
@@ -155,7 +155,7 @@
 	icon_state = "milosoup"
```
Comment (TheVekter) [inline code/game/objects/items/food/soup.dm:155]: I assume this is supposed to be miso? Why the fuck is it milo?

Comment (ShibaInuLord) [reply code/game/objects/items/food/soup.dm:155]: milo soup

**Comment by maxymax13** [comment]
add buttered toast as a mask slot item for le anime

**Comment by carshalash** [comment]
> reading "breakfast" and "japanese" gave me an idea: add buttered toast as a mask slot item for le anime

That's already a thing dude.

**Comment by maxymax13** [comment]
> 
> 
> > reading "breakfast" and "japanese" gave me an idea: add buttered toast as a mask slot item for le anime
> 
> That's already a thing dude.

shitting the one time my brain cells ding together and i've been bested

**Comment by carshalash** [comment]
inb4 this gets closed too because oranges is bored.

**Comment by SweptWasTaken** [comment]
Can you add Okonomiyaki?

https://youtu.be/DNg7Obo-DDM

**Comment by carshalash** [comment]
> Can you add Okonomiyaki?
> 
> https://youtu.be/DNg7Obo-DDM

If someone sprite okonomiyaki i'll make a recipe and balance it to be a decent meal! It's not really breakfast food though.
Also, how dare you post binging with babish at me.

**Comment by Mothblocks** [comment]
what is milo soup? we only have miso soup

**Pull request merged**